scripts/intfutil

```diff
@@ -23,42 +25,75 @@ PORT_DESCRIPTION = "description"
 PORT_OPTICS_TYPE = "type"
 PORT_PFC_ASYM_STATUS = "pfc_asym"
 
+def db_connect_configdb():
+    """
+    Connect to configdb
+    """
+    config_db = ConfigDBConnector()
+    if config_db is None: 
+        return None 
+    config_db.connect()
+    return config_db 
+
+
+def get_interface_vlan_dict(config_db):
+    """
+    Get info from REDIS ConfigDB and create interface to vlan mapping
+    """
+    get_int_vlan_configdb_info = config_db.get_table('VLAN_MEMBER')
+    int_list = []
+    vlan_list = []
+    for line in get_int_vlan_configdb_info:
+        vlan_number = line[0]
+        interface = line[1]
+        int_list.append(interface)
+        vlan_list.append(vlan_number)
+    int_to_vlan_dict = dict(zip(int_list, vlan_list))
+    return int_to_vlan_dict
+
+
+def config_db_vlan_port_keys_get(int_to_vlan_dict, intf_name):
+    """
+    Get interface vlan value and return it.
+    """
+    vlan = "routed"
+    if intf_name.startswith("Ethernet"):
```

Comment:
I'm not sure what you want instead.  The code below in the class if like this: 

class IntfStatus(object):

    def display_intf_status(self, appl_db_keys, portchannel_speed_dict):
        """
            Generate interface-status output
        """

        i = {}
        table = []
        key = []

        #
        # Iterate through all the keys and append port's associated state to
        # the result table.
        #
        for i in appl_db_keys:
            key = re.split(':', i, maxsplit=1)[-1].strip()
            if key and key.startswith('Ethernet'):
         
So I was just trying to match up with what was already built there.  Since when it goes through the class it's only looking for Ethernet and then I added a section for PortChannel for this change.  Can you suggest what you would be expecting here?

Reply:
fixed